### ISSUE TITLE
Duplicate chromium in 03-desktop

### ISSUE THREAD

**Issue by blabno** [Tomas-M/linux-live#275]
I tried building ISO without apps and chromium 
```
genslaxiso -e 'apps|chromium' slax-no-apps.iso
```
But `chromium` is still available when running launcher via `fbappselect`.

**Comment by crim50n** [comment]
The Chromium shortcut just allows you to download and install Chromium, it should be present in 03-desktop. If you want to remove the shortcut, you will have to rebuild the module yourself.

**Comment by blabno** [comment]
Is it possible with `savechanges` to record files removal?

**Comment by equetzal** [comment]
@blabno, yes. AUFS support file removals by adding a whiteout file. Just make sure to include that file in your module.
